module5/intermission/index.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: page
+title: Intermission Work
+---
+
+Before starting Module 5, spend a minimum of 2 hours working through these [PostgreSQL Exercises](https://pgexercises.com/).
```

Comment:
I think it might be helpful to give some context here about why we are coming back to SQL after so long away from it. 

Something about how your applications won't use SQL to interact with the DB, but backend developers often do. And as we are preparing for interviews, SQL is something that could definitely be part of a BE interview, so we're going to cycle back in SQL practice.